guides/common/modules/proc_viewing-multiple-content-view-environment-activation-keys.adoc

```diff
@@ -0,0 +1,38 @@
+[id="Viewing_multiple_content_view_environment_activation_keys_{context}"]
+= Viewing multiple content view environment activation keys
+
+Use this procedure to view activation keys with multiple content view environments or multi-environment activation keys in the {ProjectWebUI} and Hammer CLI.
```

Comment:
```suggestion
Use this procedure to view activation keys with multiple content view environments, or multi-environment activation keys, in the {ProjectWebUI} and Hammer CLI.
```

Comma makes a difference to meaning here! (also, should multi-environment activation keys be first?)

Reply:
I'll put "multi-environment activation key"s first. Commas aren't necessary since the phrase "or multi-environment activation keys" is essential to the sentence.